.gitignore

```
@@ -24,4 +24,8 @@ bin/main/application.yaml
 
 applicationinsights-agent-*.jar
 *.log
-bin/migration/failed_imports_log.txt
+bin/migration/migration_reports/failed_imports_log.txt
+bin/migration/migration_reports/migration_summary_logs.txt
+bin/migration/__pycache__/
+bin/migration/tables/__pycache__/
+bin/migration/migration_reports/__pycache__
```

This could probably be simplified
```suggestion
bin/migration/*/*.txt
bin/migration/**/__pycache__/
```